projects/mac_ai/testing/ramalama.py

```
@@ -91,7 +91,12 @@ def build_container_image(base_work_dir, ramalama_path):
         )
 
         if config.project.get_config("prepare.ramalama.build_image.name") == "remoting":
-            extra_env["LLAMA_CPP_PULL_REF"] = config.project.get_config("prepare.llama_cpp.source.repo.version")
+            version = config.project.get_config("prepare.llama_cpp.source.repo.version")
+            if version.startswith("pr-"):
+                pr_version = version.removeprefix("pr-")
```

Ruff (0.12.2)

Local variable `pr_version` is assigned to but never used

Remove assignment to unused variable `pr_version`

(F841)